custom-vu/src/main/kotlin/jces1209/vu/action/WorkOnSearch.kt

```diff
@@ -54,6 +59,69 @@ class WorkOnSearch(
         probability: Float
     ): Boolean = (random.random.nextFloat() < probability)
 
+    private fun switchViews() {
+        issueNavigator
+            .openNavigator()
+            .waitForNavigator()
+        meter.measure(
+            key = MeasureType.SWITCH_VIEWS,
+            action = {
+                meter.measure(
+                    key = MeasureType.SWITCH_VIEWS_CHANGE_VIEW_POPUP,
+                    action = {
+                        issueNavigator.changeViewPopup()
+                    }
+                )
+                val viewType = issueNavigator.getViewType()
+                if (viewType == IssueNavigator.ViewType.DETAIL) {
+                    meter.measure(
+                        key = MeasureType.SWITCH_VIEWS_CHANGE_VIEW_TYPE,
+                        action = {
+                            issueNavigator.changeViewType(IssueNavigator.ViewType.LIST)
+                            issueNavigator.waitForNavigator()
+                        }
+                    )
+                    meter.measure(
+                        key = MeasureType.SWITCH_VIEWS_CHANGE_VIEW_POPUP,
+                        action = {
+                            issueNavigator.changeViewPopup()
+                        }
+                    )
+                    meter.measure(
+                        key = MeasureType.SWITCH_VIEWS_CHANGE_VIEW_TYPE,
+                        action = {
+                            issueNavigator.changeViewType(IssueNavigator.ViewType.DETAIL)
+                            issueNavigator.waitForNavigator()
+                        }
+                    )
+                } else if (viewType == IssueNavigator.ViewType.LIST) {
+                    meter.measure(
```

Comment:
it just necessary to change to another view
e.g.
if current view == LIST
then change to DETAIL
and measure it